src/renderer/components/item-object-list/item-list-layout.tsx

```diff
@@ -215,8 +209,7 @@ export class ItemListLayout extends React.Component<ItemListLayoutProps> {
         filterItems.push(filterCallback);
       }
     });
-
-    const items = this.props.items ?? allItems;
+    const items = this.props.items ?? this.props.store.items;
 
     return this.applyFilters(filterItems, items);
   }
```

Comment:
We shouldn't be pushing to a prop. Plus I think that `filterItems` will soon going away. However, here is version that doesn't

```typescript
  @computed get items() {
    const { filters, filterCallbacks } = this;
    const filterGroups = groupBy<Filter>(filters, ({ type }) => type);

    const filterItems = Object.keys(filterGroups)
      .map(type => [filterCallbacks[type], filterGroups[type]] as const)
      .filter(([callback, groups]) => callback && groups.length)
      .map(([callback]) => callback)
      .concat(this.props.filterItems);

    const items = this.props.items ?? this.props.store.items;

    return this.applyFilters(filterItems, items);
  }
```

Reply:
Oh, forgot this. Fixed